### ISSUE TITLE
Short vcard icon compared to others

### ISSUE THREAD

**Issue by denisz1** [twbs/icons#1539]
Hello. The vcard icon is ["scrunched"](https://github.com/twbs/icons/issues/1477#issuecomment-1312276399) vertically, look below for repro link (thanks to @lonix1).

It is too short against the other icons, so it looks funny.

- Operating system and version windows, android
- Browser and version chrome, edge
- A reduced test case: https://jsfiddle.net/0kz9dsro

**Comment by korki43** [comment]
I don't think this is a problem. Business cards are usually oriented horizontally. Making it as tall as the other icons would result in a square or vertical business card which would look way worse IMO.

**Comment by korki43** [comment]
https://codepen.io/korki43/pen/ExeMXBv

As you can see, even the FontAwesome vcard icon is a bit shorter than the other icons.
